modules/apps/product-navigation/product-navigation-control-menu-web/src/main/resources/META-INF/resources/css/_TabItem.scss

```diff
@@ -1,133 +1,135 @@
-	margin-bottom: 4px;
-	padding: 0 4px 0 12px;
-
-	.grid & {
-		border: 1px solid $gray-300;
-		flex-wrap: wrap;
-		margin-bottom: 8px;
-		padding: 0;
-		width: calc(50% - 4px);
-
-		&-body {
-			flex-wrap: wrap;
-			text-align: left;
-			width: 100%;
+html#{$cadmin-selector} {
+	.cadmin {
+		.sidebar-body__add-panel__tab-item {
+			align-items: center;
+			border-radius: 4px;
+			color: $cadmin-secondary-d2;
+			cursor: grab;
+			display: flex;
+			height: 32px;
+			margin-bottom: 4px;
+			padding: 0 4px 0 12px;
+
+			&.sidebar-body__add-panel__tab-portlet-item {
+				margin-left: 16px;
+			}
 
-			.icon {
-				border-bottom: 1px solid $gray-300;
-				font-size: 40px;
-				padding: 4px 0 18px;
-				text-align: center;
-				width: 100%;
+			&.multiline {
+				height: 50px;
+				position: relative;
 
-				.lexicon-icon {
-					fill: $secondary-l1;
-					margin-right: 0;
+				.sidebar-body__add-panel__tab-item-add {
+					position: absolute;
+					right: 4px;
+					top: 0;
 				}
 			}
 
-			.text {
-				padding: 4px 8px 0;
-				width: 100%;
-			}
-		}
+			&-add {
+				font-size: 14px;
+				opacity: 0;
 
-		&.multiline {
-			height: 8.5rem;
+				&:focus {
+					opacity: 1;
+				}
 
-			.sidebar-body__add-panel__tab-item-add {
-				right: 0;
+				.lexicon-icon {
+					fill: $cadmin-gray-500;
+				}
 			}
-		}
-	}
 
-	&.sidebar-body__add-panel__tab-portlet-item {
-		margin-left: 16px;
-	}
+			&:focus,
+			&:hover {
+				background-color: $cadmin-gray-100;
 
-	&.multiline {
-		height: 3.125rem;
-		position: relative;
+				.sidebar-body__add-panel__tab-item-add {
+					opacity: 1;
+				}
+			}
 
-		.sidebar-body__add-panel__tab-item-add {
-			position: absolute;
-			right: 4px;
-			top: 0;
-		}
-	}
+			&.disabled {
+				color: $cadmin-secondary-l1;
+				cursor: default;
 
-	&-add {
-		font-size: 0.875rem;
-		opacity: 0;
+				&:hover {
+					background-color: transparent;
+				}
+			}
 
-		&:focus {
-			opacity: 1;
-		}
+			&-body {
+				align-items: center;
+				display: flex;
+				font-size: 14px;
+				text-align: center;
+				width: calc(100% - 24px);
 
-		.lexicon-icon {
-			fill: $gray-500;
-		}
-	}
+				.icon {
+					align-self: flex-start;
+					width: 30px;
+				}
 
-	&:focus,
-	&:hover {
-		background-color: $gray-100;
+				.lexicon-icon {
+					margin-right: 16px;
+					transform: translateY(1px);
+				}
 
-		.sidebar-body__add-panel__tab-item-add {
-			opacity: 1;
-		}
-	}
+				.text {
+					flex: 0 1 100%;
+					overflow: hidden;
+					text-align: left;
+				}
 
-	&.disabled {
-		color: $secondary-l1;
-		cursor: default;
+				.title {
+					font-size: 12px;
+					font-weight: 600;
+				}
 
-		&:hover {
-			background-color: transparent;
+				.subtitle {
+					color: $cadmin-gray-600;
+					flex-basis: 100%;
+					font-size: 12px;
+				}
+			}
 		}
-	}
 
-	&-body {
-		align-items: center;
-		display: flex;
-		font-size: 0.875rem;
-		text-align: center;
-		width: calc(100% - 1.5rem);
+		.grid .sidebar-body__add-panel__tab-item {
```

Comment:
Why are we preferring to give priority to the "parent" instead of seeing it as a variant?

- before: "when `.sidebar-body__add-panel__tab-item` is inside `.grid` it behaves in a way"
- now: "when `.grid` has `.sidebar-body__add-panel__tab-item` as a child, the second one behaves in a way"

Don't get me wrong, I find the proposal very useful in some context (e.g.: a "grid" module that takes this CSS and adapt it) but not sure what advantages it gives us here 🤔

Reply:
@edalgrin the code `.grid &` takes `.grid` and places it outside `.sidebar-body__add-panel__tab-item`. It would output `.grid .sidebar-body__add-panel__tab-item {}`. I just rewrote it to make it work with cadmin. If I didn't it would output `.grid html:not(_):not(__) .cadmin .sidebar-body__add-panel__tab-item`.